sdk/storage/storage-blob/package.json

```diff
@@ -71,6 +71,8 @@
     "build:browserzip": "gulp zip",
     "build:es6": "tsc -p tsconfig.json",
     "build:nodebrowser": "rollup -c 2>&1",
+    "build:js-samples": "npm run clean && npm run build:es6 && npm run build:nodebrowser",
+    "build:ts-samples": "npm run clean && cd samples && tsc -p . ",
```

Comment:
For debugging the samples, `build:js-samples` & `build:ts-samples` are being used as prelaunch tasks in VSCode Debug config.
Is the nomenclature `build:js-samples` fine ? 


Or something like `build-samples:js` & `build-samples:ts`?
(to be in sync with the alphabetical order of the scripts in package.json and still keep both of them together)

Reply:
Those script names are fine, the build system doesn't care about any aside from the ones that are in the documentation. Personally I like those names.

I'm mildly curious why the build:js-samples seems to do the exact same thing as the regular "build" script (except it doesn't generate a zipfile).

Reply:
@bsiegel 


    "build:es6": "tsc -p tsconfig.json",
    "build:nodebrowser": "rollup -c 2>&1",

- We import rolled-up src code in the js samples. 
- So, to execute/debug a sample, we need the rolled-up source code which is generated by `npm run build:nodebrowser`, 
- [ input for rollup is `tsc` compiled src code, which is obtained from `npm run build:es6` ]


https://github.com/Azure/azure-sdk-for-js/blob/5ae75642abb1cc73bd785e430624607b3582c159/sdk/storage/storage-blob/package.json#L74
In the command `build:js-samples`, instead of doing `npm run build:nodebrowser`, we can just do `cross-env ONLY_NODE=true rollup -c 2>&1` and successfully execute & debug the samples.